frontend/src/components/LinkCreation.jsx

```diff
@@ -160,6 +162,43 @@ export class LinkForm extends React.Component {
     }, 1000);
   }
 
+  renderSuggestedLinks() {
+    const { suggestedLinks, goSupportedInCurrentSession } = this.props
+    if (!suggestedLinks.length) {
+      return;
+    }
+
+    return <div style={{fontSize: '0.9em', marginTop: '20px'}}>
+      Did you mean any of this similar go links?
+      {suggestedLinks.map(({id, namespace, shortpath, destination_url}) => {
+        const host = !goSupportedInCurrentSession
```

Comment:
duplicated this code to avoid changes not related to this pull request, if required can move it to `/utils.js` and reuse for success message